source/SAMRAI/hier/PatchData.h

```diff
@@ -160,6 +169,12 @@ class PatchData
       const PatchData& src,
       const BoxOverlap& overlap) = 0;
 
+   virtual void
+   copy(
+      const PatchData& src,
+      const BoxOverlap& overlap,
+      tbox::KernelFuser& fuser);
```

Comment:
As it stands I'll have to implement both if I want to use fusion. I guess I can have one implementation for both that takes the fuser pointer and use or not use it under an abstraction layer to keep things single source. I'll need to use some macros to maintain support for older versions of samrai but that was pretty much inevitable.